vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -11779,9 +11779,31 @@ quantitykind:PhotonRadiance
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Photon Radiance"@en ;
 .
+quantitykind:PhotosyntheticPhotonFluxDensity
+  rdf:type qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A1E0L-2I0M0H0T-1D0 ;
+  qudt:plainTextDescription "Photosynthetically Active Radiation are the wavelengths of light within the visible range of 400 to 700 nanometers (nm) that are critical for photosynthesis. PPFD measures the amount of PAR light (photons) that arrive at the plant’s surface each second. The PPFD is measured at various distances with a Full-spectrum Quantum Sensor, also known as a PAR meter. Natural sunlight has a PAR value of 900-1500μMol/m2/s when the sun is directly overhead. For a grow light to be effective, it should have PAR values of 500-1500 μMol/m2/s." ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
+  rdfs:label "Photosynthetic Photon Flux Density" ;
+  prov:wasDerivedFrom <https://www.dormgrow.com/par> ;
+  qudt:informativeReference "https://www.gigahertz-optik.com/en-us/service-and-support/knowledge-base/measurement-of-par/"^^xsd:anyURI ;
+  skos:altLabel "PPFD" ;
+  qudt:applicableUnit unit:MicroMOL-PER-M2-SEC ;
```

Comment:
Thanks for entering this. However, the values for qudt:applicableUnit are auto-generated, as described [here](https://github.com/qudt/qudt-public-repo/wiki/Advanced-User-Guide#4-computing-applicable-units-for-a-quantitykind), where it mentions "Using the above algorithm, the values for qudt:applicableUnit are inferred and asserted in the Quantity Kind graph."

 I recognize this is not obvious, but the key thing to add (which you did) is the link from the Unit to the QuantityKind.
The next time we publish the web pages, the applicable unit values will be there, as per the algorithm.

Reply:
noted, thanks ;) btw, what is the convention used to represent dimension label, e.g., A1E0L0I0M0H0T-1D0. Is tit documented else where? it will help avoid adding duplicate dimensions in future.

Reply:
The convention is that the rdfs:label for the dimension vector is equal to the local name of the URI. You are correct that it's not mentioned in [the documentation](https://github.com/qudt/qudt-public-repo/wiki/DimensionVector-Vocabulary-Submission-Guidelines). We'll have to fix that.